### PR TITLE
feat: #6477 analytics filters

### PR DESCRIPTION
**Pull request by willmcvay**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #6477

### CODE REVIEW

File: packages/developer-portal/src/components/pages/analytics-v2/calculator/index.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React, { FC } from 'react'
+import { Title } from '@reapit/elements'
+
+export const AnalyticsCalculator: FC = () => {
+  return <Title>Calculator</Title>
+}
+
+export default AnalyticsCalculator
```
Comment (bashleigh) [inline packages/developer-portal/src/components/pages/analytics-v2/calculator/index.tsx:8]: Is there a reason for the double non-named export?

File: packages/developer-portal/src/components/pages/analytics-v2/calls/index.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React, { FC } from 'react'
+import { Title } from '@reapit/elements'
+
+export const AnalyticsCalls: FC = () => {
+  return <Title>Calls</Title>
+}
+
+export default AnalyticsCalls
```
Comment (bashleigh) [inline packages/developer-portal/src/components/pages/analytics-v2/calls/index.tsx:8]: again, reason for the double unnamed export?

File: packages/developer-portal/src/components/pages/analytics-v2/costs/index.tsx [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import React, { FC } from 'react'
+import { Title } from '@reapit/elements'
+
+export const AnalyticsCosts: FC = () => {
+  return <Title>Costs</Title>
+}
+
+export default AnalyticsCosts
```
Comment (bashleigh) [inline packages/developer-portal/src/components/pages/analytics-v2/costs/index.tsx:8]: And again

File: packages/developer-portal/src/components/pages/analytics-v2/index.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+import { AnalyticsPage } from './page'
+import { AnalyticsProvider } from './state/use-analytics-state'
+
+export const AnalyticsV2: FC = () => (
+  <AnalyticsProvider>
+    <AnalyticsPage />
+  </AnalyticsProvider>
+)
+
+export default AnalyticsV2
```
Comment (bashleigh) [inline packages/developer-portal/src/components/pages/analytics-v2/index.tsx:11]: and again

File: packages/developer-portal/src/components/pages/analytics-v2/installations/index.tsx [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import React, { FC } from 'react'
+import { Title } from '@reapit/elements'
+
+export const AnalyticsInstallations: FC = () => {
+  return <Title>Installations</Title>
+}
+
+export default AnalyticsInstallations
```
Comment (bashleigh) [inline packages/developer-portal/src/components/pages/analytics-v2/installations/index.tsx:8]: another one

**Review by joshbalfour — APPROVED** [review]
only thing I'd say is the default & named export thing is probably not needed(?)

**Review by bashleigh — CHANGES_REQUESTED** [review]

**Review by bashleigh — APPROVED** [review]

**Pull request merged**